### PR TITLE
Aug 2024 lightning talk titles (draft) and link to my slides

### PR DESCRIPTION
**Pull request by jberthold**
Maybe people can still modify the talk titles as they see fit? 
Otherwise proposing to merge as-is.

### CODE REVIEW

File: 2024/README.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ Talks given and offers of talks for 2024. Please PR with your slides or talk off
  - Jost Berthold, A Tour of GHC's Garbage Collection [slides](./FPSyd202407_GHCGarbageCollection.pdf)
 
 ## August
- - ???
+ - Lightning talks
+   - Burin C, Closure-scripting all the things
```
Comment (tmcgilchrist) [inline 2024/README.md]: I managed to grab these references from Burin's talk, dumping here while they are fresh:
Clojure - Babashka and GraalVM
https://github.com/burinc/viip
https://github.com/hyperfiddle/electric
Jank Language
Hy Language
Rama Clojure

Comment (jberthold) [reply 2024/README.md]: @burinc is the title approved? I can add the links behind the title, or you can suggest changes here.

Comment (burinc) [reply 2024/README.md]: Thanks @jberthold @tmcgilchrist for a great event.
If you you can add more links for those who may want to learn more about Clojure.

My talk title maybe `Automate all the things with Clojure/Babashka`

- [Babashka](https://babashka.org)
- [Deep Clojure/Python Integration](https://github.com/clj-python/libpython-clj)
- [python + clojure - examples](https://github.com/gigasquid/libpython-clj-examples)
- [Python + Clojure - OpenCV](https://github.com/gigasquid/libpython-clj-examples/tree/master/src/gigasquid/opencv)
- [https://hylang.org](https://hylang.org)
- [Jank](https://jank-lang.org)
- [Rama Clojure](https://blog.redplanetlabs.com/2023/10/11/introducing-ramas-clojure-api/)
- [viip - print source code with Clojure + GraalVM + Vim](https://github.com/burinc/viip)

Comment (jberthold) [reply 2024/README.md]: Thanks. Maybe the additional links can go to a small file of their own to keep this overview concise.
```suggestion
   - Burin C, Automate all the things with Clojure/[Babashka](https://babashka.org/). ([additional links](./closure-links-20240828.md))
```

File: 2024/README.md [hunk 2]
```diff
@@ -24,7 +24,11 @@ Talks given and offers of talks for 2024. Please PR with your slides or talk off
  - Jost Berthold, A Tour of GHC's Garbage Collection [slides](./FPSyd202407_GHCGarbageCollection.pdf)
 
 ## August
- - ???
+ - Lightning talks
+   - Burin C, Automate all the things with Clojure/[Babashka](https://babashka.org/). ([additional links](./closure-links-20240828.md))
+   - Jonathan Merritt, Using ELM for interactive animated SVG-pictures on the web
+   - Jost Berthold, [Haskell best practices - an invitation to discuss](https://docs.google.com/presentation/d/1VfJ3E_NBv0NDFqDS_csqskSXmQ3paWJyPSNrWH6UWLg/edit#slide=id.g2836b6e9e84_0_114)
+   - Tim McGilchrist, Adding source links for debugging to OCaml's generated assembler code
```
Comment (tmcgilchrist) [inline 2024/README.md]:
```suggestion
   - Tim McGilchrist, Adding source links for debugging to OCaml's generated assembler code, blogged about it here [Building OCaml from Assembly](https://lambdafoo.com/posts/2024-08-30-building-ocaml-from-assembly.html)
```

**Pull request merged**